src/test/java/com/epam/tc/steps/ActionSteps.java

```diff
@@ -0,0 +1,70 @@
+package com.epam.tc.steps;
+
+import com.epam.tc.Exceptions.NotFoundLabelFromBoardException;
+import com.epam.tc.dto.BoardDto;
+import com.epam.tc.dto.LabelDto;
+import com.epam.tc.dto.ListDto;
+import com.epam.tc.service.TrelloService;
+import com.epam.tc.tests.TrelloFunctionTests;
+import java.util.List;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+
+public class ActionSteps {
+
+    public ActionSteps createBoard(String boardName) {
+        TrelloFunctionTests.boardDto = TrelloService.createBoardByName(boardName);
+        TrelloFunctionTests.boardDtosListToDelete.add(TrelloFunctionTests.boardDto);
```

Comment:
There there will be some problems, if there are more than 1 class with tests, so objects from particular test class shouldn't be used at steps level